package.json

```diff
@@ -73,6 +73,6 @@
     "fmt": "prettier --write .",
     "lint": "tsc && eslint \"src/**/*.{js,ts,tsx}\" --quiet --fix",
     "build": "tsup",
-    "prepare": "husky"
+    "prepare": "husky install"
```

Comment:
why did we add install back? I was getting a warning that install is deprecated in qstash-js

Reply:
it's only for husky@9 and later